poker/utils/constants.py

```diff
@@ -1,22 +1,40 @@
 from enum import Enum
 
+import emoji
 
-SUITS = ["hearts", "diamonds", "spades", "clubs"]
-FACE_CARDS = "J Q K A".split()
+SUITS = f"{emoji.emojize(':heart_suit:')} {emoji.emojize(':diamond_suit:')} {emoji.emojize(':spade_suit:')} {emoji.emojize(':club_suit:')}".split()
 NUMBER_CARDS = list(range(2, 11))
```

Comment:
might not need the list() here?

Reply:
I removed the list and didn't work. I suppose I could remove it here and then update the deck module.